### DISCUSSION TITLE
do you guys also support Bicep

### DISCUSSION THREAD

**Discussion by say2merohit** [microsoft/NubesGen#174] [Q&A]
We are starting our journey in Azure with Bicep.  Do you guys offer similar thing with bicep as well  ?

**Comment by jdubois** [comment]
This is ticket #164 and this is being worked on by @cmaneu 
So isn't supported yet, but should be soon. Contributions and help is also welcome if you want.

Please also have a look at our CoC at https://github.com/microsoft/NubesGen/blob/main/CODE_OF_CONDUCT.md and note that this would work for anyone, and not only "guys".

**Comment by cmaneu** [comment]
Hey @say2merohit, 
I'm indeed working on adding the bicep support.
What services are a priority for you?

**Comment by say2merohit** [comment]
Wow  it’s an honour for me to be able to communicate with you Chris you are Cool …
Great videos with azure devops …What we are really looking for is Network , App services , Container , VMs I know it’s a lot to ask for but these are the options we are looking forward to ..
If we get a starting point then we can build upon it .

Regards,
Rohit

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
